.github/workflows/build_native_linux_packages.yml

```diff
@@ -0,0 +1,131 @@
+name: Build Native Linux Packages
+
+on:
+  workflow_call:
+    inputs:
+      amdgpu_family:
+        description: gfx arch for creating the artifact fetch url
+        required: true
+        type: string
+      artifact_run_id:
+        description: workflow run id to download the artifacts from
+        type: string
+      rocm_version:
+        description: ROCm version to append to the package( Like 8.0.0 or 8.1.0).
+        required: true
+        type: string
+      package_type:
+        description: Specify whether debian or rpm packages are needed (deb or rpm).
+        required: true
+        type: string
+      package_suffix:
+        description: The suffix to be added to package name(build_no or master or rc or combiantion).
+        required: false
+        type: string
+  workflow_dispatch:
+    inputs:
+      amdgpu_family:
+        type: choice
+        options:
+          - gfx110X-dgpu
+          - gfx1151
+          - gfx120X-all
+          - gfx94X-dcgpu
+          - gfx950-dcgpu
+        default: gfx94X-dcgpu
+      artifact_run_id:
+        description: workflow run id to download the artifacts from
+        type: string
+        default: "17840675558"
+      rocm_version:
+        description: ROCm version to append to the package( Like 8.0.0 or 8.1.0)
+        type: string
+        default: "7.9.0"
```

Comment:
Usually, when doing ad-hoc builds with a default, I use a version that is clearly invalid so it can never sort with current versions (i.e. 0.0.1 or something).